### DISCUSSION TITLE
Change the category of a discussion after the fact

### DISCUSSION THREAD

**Discussion by ebouchut** [community/community#2997] [Discussions] [answered]
(Provided a discussion has no answer yet) it would be useful to have the ability to move it to another category.
Say I started a discussion in the "General" category.
I would like to be able to change its category and move it to the "Ideas" category for instance.

**Comment by becca** [comment, marked as answer]
You can move a discussion into another category in a repo. See https://github.com/github/feedback/discussions/2913#discussioncomment-181893
